package-lock.json

```diff
@@ -1,78 +1,95 @@
 {
   "name": "@diplodoc/tabs-extension",
-  "version": "2.0.8",
-  "lockfileVersion": 3,
+  "version": "2.0.9",
+  "lockfileVersion": 1,
```

Comment:
You should use node version from project's `nvm`

Reply:
There is no `.nvmrc`.
This is also a problem of some last fixes, that plugin uses node version incompatible with core tech.

Reply:
Got it. I regenerated `package-lock` for nodejs=14

Reply:
And I added `.nvnrc`